crates/cargo-wdk/src/actions/package/error.rs

```diff
@@ -0,0 +1,76 @@
+use std::{path::PathBuf, string::FromUtf8Error};
+
+use thiserror::Error;
+
+use crate::{actions::build::BuildActionError, providers::error::CommandError};
+
+/// Errors for the package project action layer
+#[derive(Error, Debug)]
+pub enum PackageProjectError {
```

Comment:
Might be better to rename this to `PackageActionError`. The corresponding error types for new and build actions are named simiiarly.